src/game.js

```diff
@@ -9,16 +10,24 @@ class Game {
 	static create({cols = 10, rows = 10} = {}) {
 		const game = new Game({cols,rows});
 		game.id = dbGame.length + 1;
-		game.playerId = idHelper();
+		game.playerIdOne = idHelper();
 		const token = idHelper();
 		game.token = token
 		dbGame.push(game);
+
 		game.session = `http://localhost:3000/game?token=${token}`;
 
+		gameDB.sync({force: false})
+		.then(function () {
+			return gameDB.create({
+				playerIdOne: game.playerIdOne,
+				token: game.token});
+		});
+
 		return Promise.resolve({
 			id : game.id, 
 			session : game.session,
-			playerId : game.playerId
+			playerIdOne : game.playerIdOne
 		})

```

Comment:
you should chain all the promises, otherwise you'll get unexpected results, for example what will happen if game.create() fails?